pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -289,20 +289,15 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 		qosContainerManager: qosContainerManager,
 	}
 
-	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.TopologyManager) {
-		cm.topologyManager, err = topologymanager.NewManager(
-			machineInfo.Topology,
-			nodeConfig.ExperimentalTopologyManagerPolicy,
-			nodeConfig.ExperimentalTopologyManagerScope,
-			nodeConfig.ExperimentalTopologyManagerPolicyOptions,
-		)
-
-		if err != nil {
-			return nil, err
-		}
+	cm.topologyManager, err = topologymanager.NewManager(
+		machineInfo.Topology,
+		nodeConfig.ExperimentalTopologyManagerPolicy,
+		nodeConfig.ExperimentalTopologyManagerScope,
+		nodeConfig.ExperimentalTopologyManagerPolicyOptions,
```

Comment:
sorry for late notice: should we rename these fields dropping the `Experimental` prefix? I think we should

/cc @SergeyKanzhelev

Reply:
No worries, I can add a commit for this.